extensions/MyDashboard/web/js/query.js

```diff
@@ -44,6 +46,19 @@ $(function() {
             }
         }
 
+        // Grab last used auto-refresh configuration from cookie or use default
+        var autorefresh_cookie = Y.Cookie.get("my_dashboard_autorefresh");
+        if (autorefresh_cookie) {
+            var cookie_value_found = 0;
+            if ("true" == autorefresh_cookie) {
+                Y.one("#auto_refresh").set('checked', true);
+                cookie_value_found = 1;
+            }
+            if (!cookie_value_found) {
+                Y.Cookie.set("my_dashboard_autorefresh", "false");
+            }
+        }
```

Comment:
This could all be shortened to:
```
        if (autorefresh_cookie == 'true') {
            Y.one("#auto_refresh").set('checked', true);
        }
        else {
            Y.Cookie.set("my_dashboard_autorefresh", "false");
        }
```